<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageViewer</class>
 <widget class="QMainWindow" name="ImageViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>675</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>675</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ImageViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>161</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>75</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="fileNameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>830</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="previousButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="text">
     <string>Previous</string>
    </property>
   </widget>
   <widget class="QToolButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QToolButton" name="startSlideShowButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Start SlideShow</string>
    </property>
   </widget>
   <widget class="QToolButton" name="stopSlideShowButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop SlideShow</string>
    </property>
   </widget>
   <widget class="QLabel" name="delayLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>11</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SlideShow Delay:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="delayComboBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="shuffleCheckBox">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>12</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Shuffle</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionBrowse"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPrevious"/>
    <addaction name="actionNext"/>
    <addaction name="separator"/>
    <addaction name="actionStart_SlideShow"/>
    <addaction name="actionStop_SlideShow"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionBrowse">
   <property name="text">
    <string>Browse</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="text">
    <string>Previous</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionStart_SlideShow">
   <property name="text">
    <string>Start SlideShow</string>
   </property>
  </action>
  <action name="actionStop_SlideShow">
   <property name="text">
    <string>Stop SlideShow</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
